### ISSUE TITLE
Stopping system throws an error

### ISSUE THREAD

**Issue by kgxsz** [danielsz/sente-system#2]
Hi,

When I try to stop the system, I'm seeing:

```
[server-loop] ERROR - increase :queue-size if this happens often
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@293946d9 rejected from java.util.concurrent.ThreadPoolExecutor@743088b[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 465]
```

This may be related to [this issue](https://github.com/ptaoussanis/sente/issues/15) that I also commented on. Do you have any ideas what might be causing this?

Thanks.

**Comment by danielsz** [comment]
No, I don't know what causes this error, but this occurs only at development time if I'm not mistaken, and I wouldn't worry about it. Interrupted futures have no ill effects on the runtime.

**Comment by coyotespike** [comment]
More to document than anything else, I'm seeing the same issue - a warning to increase the `queue-size`, coupled with the notification that the `FutureTask` has been rejected.

I have confirmed that this only happens when Sente is used with System. It's somewhat more pleasant not to have these warnings but as you say it doesn't seem to break anything.

```
 [clojure-agent-send-off-pool-0] ERROR - increase :queue-size if this happens often                                                                                
java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@3fb8eea3 r
ejected from java.util.concurrent.ThreadPoolExecutor@7865d9df[Terminated, pool size = 0, active 
threads = 0, queued tasks = 0, completed tasks = 155]
```

As referenced [in Sente issue 15](https://github.com/ptaoussanis/sente/issues/15).

**Comment by danielsz** [comment]
This project is way behind in terms of dependencies, and is not going to be maintained further by me. If there are any volunteers to take over, please come forward. 
I recommend referring to https://github.com/danielsz/system-websockets for wiring Sente in Clojure projects. No errors on stop, it's maintained, it's more elegant, etc.

Thank you!
